index.md

```diff
@@ -28,7 +28,7 @@ Participants join this program with a **project** that they either are already w
 
 *Please register on Open Review before July 1, 2022 to allow activation of your Open Review profile as described in the [OLS-6 application guidelines and templates](https://github.com/open-life-science/application-forms).*-->
```

Comment:
The correct word should be "Applications for OLS-7 will open at the end of 2022."
not
"Applications for OLS-7 will open ending of 2022."